From 365222dbb288ab3c81f856709bde549c79bfc844 Mon Sep 17 00:00:00 2001
From: Ricardo Cerqueira <ricardo@cyngn.com>
Date: Wed, 2 Dec 2015 16:34:22 +0000
Subject: [PATCH 05/74] libfm_jni: Do not come here unless QCOM_HARDWARE is in
 use

This and the non-QCOM (Google) FM app are mutually exclusive

Change-Id: I41f33b6d21b56a46588d696ec286ffa2d45f1f0c

libfm_jni: Use correct QCOM HW flag

BOARD_USES_QCOM_HARDWARE is common flag for QCOM devices

Change-Id: Ie5c784288ebdecf00220fe0196ba3fdd2752a1f9
---
 libfm_jni/Android.mk | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/libfm_jni/Android.mk b/libfm_jni/Android.mk
index 2ffdafd..fd839c8 100644
--- a/libfm_jni/Android.mk
+++ b/libfm_jni/Android.mk
@@ -1,3 +1,4 @@
+ifeq ($(BOARD_USES_QCOM_HARDWARE),true)
 ifneq (,$(filter $(QCOM_BOARD_PLATFORMS),$(TARGET_BOARD_PLATFORM)))
 LOCAL_PATH := $(call my-dir)
 
@@ -36,3 +37,5 @@ LOCAL_MODULE := libfmjni
 #include $(BUILD_SHARED_LIBRARY)
 
 endif
+
+endif # QCOM_HARDWARE
-- 
2.17.1

